Shaders/PBR/private/ComputeIrradianceMap.psh

```diff
@@ -8,14 +8,23 @@
 #   define NUM_THETA_SAMPLES 32
 #endif
 
+#ifndef IRRADIANCE_MAP_DIMENSION
+#   define IRRADIANCE_MAP_DIMENSION 16
+#endif
+
 TextureCube  g_EnvironmentMap;
 SamplerState g_EnvironmentMap_sampler;
 
 void main(in float4 Pos      : SV_Position,
           in float3 WorldPos : WORLD_POS,
           out float4 Color   : SV_Target)
 {
-    float3 N     = normalize(WorldPos);
+
+    uint3 TextureDimension;
+    g_EnvironmentMap.GetDimensions(0, TextureDimension.x, TextureDimension.y, TextureDimension.z);    
+    float EnvLod = max(0.0, log2(float(max(TextureDimension.x, TextureDimension.y) / IRRADIANCE_MAP_DIMENSION)) - 1.0);
```

Comment:
This is not the right way to select LOD.
A better way is to use solid angle, like we discussed offline.